manifests/init.pp

```diff
@@ -8,6 +8,8 @@
   Float   $cvmfs_quota_ratio                                     = 0.85,
   Stdlib::Absolutepath $cvmfs_cache_base                         = '/var/lib/cvmfs',
   Optional[Enum['yes','no']] $cvmfs_claim_ownership              = undef,
+  Hash $cvmfs_uid_map                                            = {},
+  Hash $cvmfs_gid_map                                            = {},
```

Comment:
Maybe add some validation to the contents of the hash.

Reply:
It's now validated as:
```
Optional[Hash[Variant[Integer,String], Integer, 1]]
```